notifier/notifier.go

```diff
@@ -221,83 +246,113 @@ func (n *Notifier) collectRogueCallbacks() {
 	}
 }
 
-// Notify posts callback info to j.CallbackURL
-func (n *Notifier) Notify(j *job.Job) error {
-	j.CallbackCount++
+// MonitorDeliveries monitors the delivery reports channel of a backend.
```

Comment:
Please document what the second argument is. Another idea (maybe better) idea would be to introduce a `BackendID` type (an alias of `string`).

Reply:
Also, please document what "monitors the delivery reports channel" means. For example:

> monitorDeliveries consumes and handles delivery reports from the running backends. See n.handleCallbackInfo()